src/hypothesis/core.py

```diff
@@ -1121,22 +1122,18 @@ def template_condition(data):
 
         if settings.verbosity == Verbosity.verbose:
             if not successful_examples[0]:
-                report(lambda: u'Trying example %s' % (
-                    nicerepr(result),
-                ))
+                report(u'Tried non-satisfying example %s' % nicerepr(result))
             elif success:
                 if successful_examples[0] == 1:
-                    report(lambda: u'Found satisfying example %s' % (
-                        nicerepr(result),
-                    ))
+                    last_repr[0] = nicerepr(result)
+                    report(u'Found satisfying example %s' % last_repr[0])
```

Comment:
The `u'...' % (thing,)` syntax is common around the project and saves us from accidentials. Let's keep that style for consistency.